develop/how-to-submit-patches/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: How to Submit Patches
+---
+
+***TL;DR:*** Please use [GitHub pull requests](https://help.github.com/articles/about-pull-requests/) or, if you can't use GitHub, send us a unified diff against the latest version of the appropriate branch (master or stable) and remember to update the docs and unit tests in any case.
+
+User feedback and, especially, patches are very important for wxWidgets. They often help us improve wxWidgets' quality and fix bugs, so we are of course happy if you contribute them (how could we dislike your help, after all?). But we have all sort of problems with applying non-standard patches. To make life easier for both you and us, please follow the few simple rules below when submitting patches.
+
+Remember that if you have any questions about the steps here, you can always post them to wx-dev mailing list and we'll do our best to help you.
+
+## Changing wxWidgets
```

Comment:
This basically duplicates (and surprisingly does not refer to) _develop/how-to-contribute/index.md_ but I suppose it is by design?

Reply:
It's just the way it historically was (with one document in the wiki and another one in Trac) and I didn't have time to change it. It's pretty clearly suboptimal though.